cms/models/pagemodel.py

```diff
@@ -354,7 +354,7 @@ def _copy_contents(self, target, language):
                 new_phs.append(ph)
                 # update the page copy
             if plugins:
-                copy_plugins_to(plugins, ph, no_signals=True)
```

Comment:
When signals are suppressed, the placeholder cache doesn't get marked dirty when plugin's are changed.